styleguide/source/_patterns/03-templates/04-topic/_topic.scss

```diff
@@ -42,6 +38,11 @@
       grid-area: 1 / 1 / 4 / 1;
       padding-left: 14px;
     }
+
+    .grid-region_content.topic_topic-articles > div {
+      padding-top: 0;
+      @include gutter($padding-bottom-full...);
```

Comment:
@designerneil almost there... if you reorder the blocks on the page (put the promo after the topic-articles block in the left sidebar, for instance), you'll see the space below the last item is too large now. Try wrapping this line in `:not(:last-child)`:

```
    .grid-region_content.topic_topic-articles > div {
      padding-top: 0;
      &:not(:last-child) {
        @include gutter($padding-bottom-full...);
      }
    }
```